src/test/ui/pattern/issue-68396-let-float-bug.rs

```diff
@@ -1,7 +0,0 @@
-fn main() {
-    let 1234567890123456789012345678901234567890e-340: f64 = 0.0;
-    //~^ ERROR could not evaluate float literal (see issue #31407)
-
-    fn param(1234567890123456789012345678901234567890e-340: f64) {}
-    //~^ ERROR could not evaluate float literal (see issue #31407)
-}
```

Comment:
Could we keep these tests as `// compile-pass` regression tests instead?

Reply:
Sure, does this look ok?

```rust
// compile-pass
// ignore-tidy-linelength
// Regression test for #31109 and #31407.

pub fn main() {
    let _: f64 = 0.3333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333;

    let _: f64 = 1234567890123456789012345678901234567890e-340;
}
```

Reply:
It does.

Reply:
I'll re-add it in (I've already rebased) and commit.

Reply:
Resolved as of https://github.com/Alexhuszagh/rust/commit/c08915141d34c75799859a983cec2b22e74065b7.